### PR TITLE
feat: Add media page

### PR DESCRIPTION
**Pull request by caseykwok**

### CODE REVIEW

File: core/src/templates/media-page.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  text-decoration: none;
+  color: #000000;
+
+  :hover {
+    color: #000000;
+  }
+`
+
+const StyledGrid = styled(Grid)`
+  margin-bottom: 16px !important;
```
Comment (mingchuno) [inline core/src/templates/media-page.js]: why so many `!important` 😢 @caseykwok

**Comment by mingchuno** [comment]
I manage to remove the `!important` by setting some extra padding/margin for `StdTitle` (which it should). And fine tune the other part to fix

**Comment by mingchuno** [comment]
LGTM

**Comment by mingchuno** [comment]
I have also applied sorted

File: core/src/templates/media-page.js (hunk 1 above)
Comment (gaplo917) [reply core/src/templates/media-page.js]: @caseykwok you need to add &&{ ... } in the styled component to override MaterialUI Component styles

Comment (caseykwok) [reply core/src/templates/media-page.js]: @gaplo917 oh!! ok, noted. thanks, will take note next time!

**Comment by caseykwok** [comment]
@mingchuno I was thinking that we'd apply sort to the query itself if possible but this will work fine as well for now

**Pull request merged**